ivy_tests/test_ivy/test_frontends/test_tensorflow/test_linalg.py

```diff
@@ -45,3 +47,40 @@ def test_tensorflow_det(
         fn_tree="linalg.det",
         input=np.asarray(x, dtype=input_dtype),
     )
+
+
+# slogdet
+@handle_cmd_line_args
+@given(
+    dtype_x=helpers.dtype_and_values(
+        available_dtypes=ivy_np.valid_float_dtypes,
+        min_value=0,
+        max_value=50,
```

Comment:
Please remove the max_value limit, as we should be testing the function exhaustively.

Reply:
done